base/applications/rapps/gui.cpp

```diff
@@ -1467,6 +1467,7 @@ class CMainWindow :
                 {
                     CDownloadManager::DownloadListOfApplications(m_AvailableApps.GetSelected());
                     UpdateApplicationsList(-1);
+                    m_ListView->SetSelected(-1, FALSE);
```

Comment:
Shouldn't it be done inside the UpdateApplicationsList function?

Reply:
Seems like I'd have to recreate the same logic as the current context to ensure that we deselect them only after installing checked packages. UpdateApplicationsList is called under many situations and installations can occur without select a package (i.e. double clicking). So it seemed more direct to just put the cleanup logic here.

But I'm new here so fine to put it wherever.

Reply:
@sanchaez or @gigaherz can help you for that :smiley: